README.md

```diff
@@ -16,6 +16,16 @@ The input to the pipeline is a standard sample sheet (passed as `--input samples
 
 The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/).
 
+## IRIDA-Next Optional Input Configuration
```

Comment:
Can you add a markdown table of the samplesheet here that includes `sample_name`, similar to the markdown table above for the minimal samplesheet?

Reply:
Sure can! Done here: [b78388a](https://github.com/phac-nml/speciesabundance/pull/24/commits/b78388a2947ac8e2659178742f49616d9999312b)